src/extractor/ConfigurationExtractor.ts

```diff
@@ -1,34 +1,63 @@
 import { countBy, toPairs, maxBy, capitalize, filter } from 'lodash';
 import { fhirtypes } from 'fsh-sushi';
+import { logger } from '../utils/GoFSHLogger';
 import { ExportableConfiguration } from '../exportable';
 
 type ImplementationGuideStatus = fhirtypes.ImplementationGuideStatus;
 
 export class ConfigurationExtractor {
   static process(resources: any[]): ExportableConfiguration {
+    const igResource = ConfigurationExtractor.getIGResource(resources);
+    const missingIGProperties: string[] = [];
+    if (igResource && !igResource.url) {
+      missingIGProperties.push('url');
+    }
+    if (igResource && !igResource.fhirVersion?.length) {
+      missingIGProperties.push('fhirVersion');
+    }
+    if (missingIGProperties.length > 0) {
+      logger.warn(
+        `ImplementationGuide missing properties needed to generate configuration file: ${missingIGProperties.join(
+          ', '
```

Comment:
I think it may be useful to add to this message to indicate that the properties are going to be inferred. Just because it may be confusing the see that the `url` is missing, but then see a `canonical` still show up in your generated config.

Reply:
Added in 20d75fa!